### ISSUE TITLE
RATING APPEAL Oneinacilian - 5.9

### ISSUE THREAD

**Issue by oneinacillian** [wax-office-of-inspector-general/waxguilds#1048]
Good Day IG's

Can you please have a look at the points we have awarded for the following:

1. Blocks Log - Please can you explain why there is no recognition for full blocks in 5.9 but there was in 5.8? For reference repository URL: https://stateblocks.oiac.io
2. State History - Please can you confirm why there is no recognition for this at all. For reference repository URL: https://stateblocks.oiac.io
3. Telegram missing/empty block bot - Please can you explain why the decay target for this offering is 0 and points awarded is only 0.5 when other similar offerings from other guilds i.e. Alohaeos have score of 1 point and decay target of 1 also. In our 5.9 update we mentioned the testnet offering in addition to main net but it did not seem to impact the score in any way.

Thank you for taking the time to review, and I await your feedback.

Best,

Cillian.

**Comment by kaefergeneral** [comment]
Hello @oneinacillian,

1. & 2. The scores for your snapshots have been added.

3. Decay targets are set based on estimates. It does not mean that the product will actually decay. We removed the decay target now, as we did not currently plan to  reduce the score.

Scores are attributed based on different metrics. The most important one being the value they add to the ecosystem. While we do see some value in the tracking of empty blocks on mainnet, we do not see value in having a second bot tracking missing rounds on testnet with a total of 8 subscribers.
In direct comparison to Aloha, their bots are established and highly utilized (104 and 86 subscribers) in our ecosystem, resulting in a total score of 1 for both of them. So we consider a score of 0.5 in comparison as justified.

Best,
your OIG
